easybuild/tools/filetools.py

```diff
@@ -1628,6 +1629,65 @@ def copy(paths, target_path, force_in_dry_run=False):
             raise EasyBuildError("Specified path to copy is not an existing file or directory: %s", path)
 
 
+def get_source_from_git(filename, targetdir, git_config):
```

Comment:
@mboisson Maybe rename to `get_source_tarball_from_git`?

Reply:
fixed in upcoming commit